app/models/intervention_type.rb

```diff
@@ -24,22 +24,14 @@
 class InterventionType < ApplicationRecord
   extend Mobility
   include TransifexSerialisable
+  include Searchable
+
   translates :name, type: :string, fallbacks: { cy: :en }
   translates :summary, type: :string, fallbacks: { cy: :en }
   translates :description, backend: :action_text
   translates :download_links, backend: :action_text
 
   include PgSearch::Model
```

Comment:
Do we still need to include this if we're not using PgSearch on this model? Same on the ActivityType

Reply:
:+1:  I've removed now